src/views/pacbio/PacbioRunShow.vue

```diff
@@ -133,6 +134,8 @@ export default {
         this.newRun()
       } else if (!this.newRecord) {
         await this.editRun(parseInt(this.$route.params.id))
+        const barcodes = this.poolBarcodes
+        await this.$store.dispatch('traction/pacbio/runCreate/findPools', { barcode: barcodes })
```

Comment:
Nice way of loading the pools for existing runs 👍

Reply:
If the pools IDs are available it may be faster to grab the pools based on their Ids but I don't think it will make much of a difference.